src/Event/AuditDeleteEvent.php

```diff
@@ -23,13 +16,17 @@ class AuditDeleteEvent implements EventInterface
      * @param string $source The name of the source (table) where the record was deleted
      * @param string $parentSource The name of the source (table) that triggered this change
      */
-    public function __construct($transactionId, $id, $source, $parentSource = null)
+    public function __construct($transactionId, $id, $source, $parentSource = null, $original = null)
     {
-        $this->transactionId = $transactionId;
-        $this->id = $id;
-        $this->source = $source;
+        if (is_array($parentSource) === true) {
```

Comment:
I would rather not have this check at all and avoid messing up with the arguments.